hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEndpointRegion.java

```diff
@@ -478,6 +517,16 @@ public void testCentralEndpointAndNullRegionFipsWithCRUD() throws Throwable {
     assertOpsUsingNewFs();
   }
 
+  /**
+   * Skip the test if the region is sa-east-1.
+   */
+  private void skipCrossRegionTest() throws IOException {
+    String region = getFileSystem().getS3AInternals().getBucketMetadata().bucketRegion();
+    if (SA_EAST_1.equals(region) || NON_AWS_REGION.equals(region)) {
```

Comment:
please change the non-aws probe for just `region == null`

Reply:
ack